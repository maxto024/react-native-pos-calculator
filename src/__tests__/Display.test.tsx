import { shallow } from 'enzyme'
import * as React from 'react'
import { NativeModules } from 'react-native'
import { Display } from '../Display'

NativeModules.UIManager = {
  measureLayoutRelativeToParent: jest.fn()
}

it('renders display correctly', () => {
  const wrapper = shallow(
    <Display
      width={400}
      height={100}
      value={'1.000'}
      style={{ color: '#000' }}
    />
  )
  expect(wrapper).toMatchSnapshot()
})
