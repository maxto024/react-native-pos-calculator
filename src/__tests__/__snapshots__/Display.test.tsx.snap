// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders display correctly 1`] = `
<Component>
  <Text
    accessible={true}
    adjustsFontSizeToFit={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    numberOfLines={1}
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
          "bottom": 0,
          "flex": 1,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        Object {
          "color": "#000",
        },
        Object {
          "color": "#000",
          "fontSize": 100,
          "height": 100,
          "lineHeight": 100,
        },
      ]
    }
  >
    1.000
  </Text>
</Component>
`;
